lib/main_containers.pm

```diff
@@ -11,18 +11,23 @@ use base 'Exporter';
 use Exporter;
 use utils;
 use version_utils;
-use main_common qw(loadtest boot_hdd_image);
 use testapi qw(check_var get_required_var get_var);
 use Utils::Architectures;
 use strict;
 use warnings;
-
+use Carp 'croak';
 
 our @EXPORT = qw(
   is_container_test
   load_container_tests
 );
 
+sub loadtest {
```

Comment:
I would make it more clear for future reviewers that we did code duplication on purpose.
```suggestion
==head2
  container_loadtest($test)
  
We duplicate c<loadtest()> function here to not have cycle dependencies between c<main_common> and this module.
==cut
sub container_loadtest {
```

And I would use a "unique" name to make it more clear that we use our own.
WDYT?